Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Replace_text_with_content_control
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream inputStream = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Load an existing Word document.
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
+                {
+                    //Add block content control into the Word document.
```

Comment:
@sharonaumao - Please modify the XML comments as we discussed.

Reply:
@Ramaraj-Marimuthu -I have done so Sir